src/Navigation/actions.js

```diff
@@ -0,0 +1,17 @@
+export const isClause = input => input === 'clause';
+export const isHeadingOne = input => input === 'heading_one';
+export const isHeadingTwo = input => input === 'heading_two';
+export const isHeadingThree = input => input === 'heading_three';
+
+const truncateHeading = (heading, length) => ((heading.length > length)
+  ? `${heading.substring(0, length)}...`
+  : heading);
+
+
+export const truncateHeader = ({ type, text }) => {
+  if (isClause(type)) return truncateHeading(text, 20);
```

Comment:
i think instead of the `if` statements and helper functions on line 1-4, this could be more simply written as a switch statement.
```
switch(type) {
    case 'clause':
      return truncateHeading(text, 20);
    case 'heading_one':
        return truncateHeading(text, 22); 
    etc, etc....
  }
```

Reply:
This is a good idea!